src/queries/eventRecords.js

```diff
@@ -102,6 +102,22 @@ export const GET_EVENT_RECORDS_AND_CATEGORIES = gql`
         kind
         addition
       }
+      contacts {
+        id
+        firstName
+        lastName
+        phone
+        email
+        fax
+        webUrls {
+          url
+          description
+        }
+      }
+      webUrls: urls {
+        url
+        description
+      }
```

Comment:
we missed someting:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/195/files